src/schema/rules/sidecars/mrs.yaml

```diff
@@ -0,0 +1,89 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+---
+# MRS Common metadata fields
+MRSScannerHardware:
+  selectors:
+    - modality == "mrs"
+  fields:
+    InstitutionName: recommended
+    InstitutionAddress: recommended
+    InstitutionalDepartmentName: recommended
+    Manufacturer: recommended
+    ManufacturersModelName: recommended
+    DeviceSerialNumber: recommended
+    StationName: recommended
+    SoftwareVersions: recommended
+    HardcopyDeviceSoftwareVersion: DEPRECATED
```

Comment:
We should not include deprecated fields in new data descriptions. If it contains information that is otherwise unspecified, then it should not be deprecated.

Reply:
Removed.